auto/cc/test

```diff
@@ -156,6 +156,21 @@ case $NXT_CC_NAME in
 
         NXT_CFLAGS="$NXT_CFLAGS -Wmissing-prototypes"
 
+        # Workaround clang bug.
+        nxt_cc_clang_major=`echo "$NXT_CC_VERSION" \
+                         | sed 's/.*version \([0-9]*\).*/\1/'`
+        nxt_cc_clang_minor=`echo "$NXT_CC_VERSION" \
+                         | sed 's/.*version [0-9]*\.\([0-9]*\).*/\1/'`
+
+        if [ "$nxt_cc_clang_major" -ge 4 ]; then
+            NXT_CFLAGS="$NXT_CFLAGS -fdeclspec"
+        else
+        if [ "$nxt_cc_clang_minor" -eq 3 ] && [ "$nxt_cc_clang_minor" -ge 8 ];
+        then
+            NXT_CFLAGS="$NXT_CFLAGS -fdeclspec"
+        fi
+        fi
```

Comment:
Hi Valentin,

Yes, I think in the long term we should actually test the flag, but since there's no script doing this in `auto/`, we will need to add a new one, which will be a bit more complex than this.

Do you agree that we should release this easy workaround now, and prepare a better solution later?

Thanks,

Alex

Reply:
`auto/feature` does the test, since it compiles the code and `NXT_TEST_CFLAGS` can be used for adding an option during the test.